ScanbotSDKExamples/MAUI/ClassicComponent.Maui/Platforms/Android/MainApplication.cs

```diff
@@ -16,7 +16,7 @@ public MainApplication(IntPtr handle, JniHandleOwnership ownership)
 
     private MauiApp CreateMauiInstance()
     {
-        Task.Run(async () =>
+        Task.Run(() =>
         {
             try
             {
```

Comment:
@scan-mayank , I believe `Task.Run` was present before because of `await Task.Delay(5000)` but now that delay is commented out. Does it make sense to run the code in a task? Maybe we can remove it. 

Or, at the very least, remove the commented out code if ti isn't needed.

Reply:
Fixed